R/set_args.R

```diff
@@ -14,6 +14,14 @@ set_args <- function(fun.name, ..., custom.config = FALSE, package='graphics'){
   config_args <- config(fun.name, custom.config = custom.config)
   user_args <- function_args(name=fun.name, package=package, ...)
   
+  if(fun.name %in% c('points', 'lines')){
```

Comment:
I think this should maybe be in function_args rather than here.  I worry that adding special cases in set_args will complicate this function beyond where it is now.